### PR TITLE
e2e: Save container logs to ARTIFACT_DIR

### PR DESCRIPTION
**Pull request by mrogers950**
This saves per-test container logs to ARTIFACT_DIR at the end of waitFor* functions.

### CODE REVIEW

File: tests/e2e/helpers.go [hunk 1]
```diff
@@ -1277,3 +1279,48 @@ func waitForCronJobWithSchedule(t *testing.T, f *framework.Framework, namespace,
...
 }
+
+func writeToArtifactsDir(t *testing.T, f *framework.Framework, scan, pod, container string, log []byte) error {
+	dir := os.Getenv("ARTIFACTS")
+	logPath := path.Join(dir, fmt.Sprintf("%s-%s-%s.log", scan, pod, container))
+	logFile, err := os.Create(logPath)
+	if err != nil {
+		return err
+	}
+	defer logFile.Close()
```
Comment (jhrozek) [inline tests/e2e/helpers.go:1290]: Please add a nosec comment here to avoid triggeric gosec.

**Comment by mrogers950** [comment]
@JAORMX @jhrozek This works now!

File: tests/e2e/helpers.go [hunk 2]
```diff
@@ -1636,3 +1640,64 @@ func getReadyProfileBundle(t *testing.T, f *framework.Framework, name, namespace
 
 	return pb, nil
 }
+
+func writeToArtifactsDir(t *testing.T, f *framework.Framework, scan, pod, container string, log string) error {
+	dir := os.Getenv("ARTIFACT_DIR")
```
Comment (jhrozek) [inline tests/e2e/helpers.go:1645]: This is a nit, but should we continue if the environmental var is not set and log to cwd or set up some other directory to put the logs to? Having the logs in cwd is slightly irritating for a local scan :-)

**Pull request merged**